rasa/nlu/extractors/crf_entity_extractor.py

```diff
@@ -307,12 +313,14 @@ def _from_crf_to_json(
             )
 
         if self.component_config["BILOU_flag"]:
-            return self._convert_bilou_tagging_to_entity_result(tokens, entities)
+            return self._convert_bilou_tagging_to_entity_result(
+                message, tokens, entities
+            )
         else:
             # not using BILOU tagging scheme, multi-word entities are split.
             return self._convert_simple_tagging_to_entity_result(tokens, entities)
 
-    def _convert_bilou_tagging_to_entity_result(self, tokens, entities):
+    def _convert_bilou_tagging_to_entity_result(self, message, tokens, entities):
```

Comment:
Same again - types and a docstring please

Reply:
fixed here https://github.com/RasaHQ/rasa/pull/4018/commits/f65f354a4d115ce52f075aed3249740b81b96601